src/main/java/cs/ualberta/cmput402/tictactoe/TicTacToeGame.java

```diff
@@ -12,6 +12,7 @@
 public class TicTacToeGame {
 
     private Board board;
+    private int[] score = {0,0,0};	// X wins, O wins, Ties
```

Comment:
According to the description of Assignment 1, Feature 2 also requires the scoreboard to record how many times that each player lose.

Reply:
Ok. That also works.